apps/cli-gui/src/app/generators/generators.routes.ts

```diff
@@ -10,7 +9,7 @@ export const GENERATOR_ROUTES: Routes = [
       {
         path: 'component',
         loadComponent: () =>
-          import('./generate-component/generate-component.component').then(
+          import('@angular-cli-gui/generators').then(
             (m) => m.GenerateComponentComponent
```

Comment:
can you rename the component to be with only one "component" suffix?

Reply:
@Danieliverant I suppose we can do it, but it would make it a bit inconsistent with other components names (like GenerateDirectiveComponent) wdyt?

Reply:
GenerateDirectiveComponent should not be Component at all.... it's a directive